client/src/components/create-assignment/LeafAssignment.js

```diff
@@ -45,11 +46,12 @@ const expiryData = {
 */
```

Comment:
I also noticed lines where spaces aren't used inside curly brackets in this file, so if you don't mind then that could be corrected. Lines 171 (`{JSON.stringify(indices) !== '[0]' ?`) and 176 (`<Box sx={{ width: '1px' }}/>}`).